sandy/njoy.py

```diff
@@ -892,6 +946,16 @@ def process(
             wdir,
             f"{outprefix}{tag}.pendf",
             )
+    if groupr:
```

Comment:
Does it work for multiple temperatures?

Reply:
I erase these options for multiple temperatures

Reply:
Then, can you add here a warning like `f"multiple temperatures were requested, GROUPR will only process the first one: {temperatures[0]:.1f} K"`

Reply:
This is still unresolved